src/test/java/com/rtb/projectmanagementtool/auth/AuthOpsTest.java

```diff
@@ -0,0 +1,125 @@
+package com.rtb.projectmanagementtool.auth;
+
+import static org.mockito.Mockito.*;
+
+import com.google.appengine.api.datastore.dev.LocalDatastoreService;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.appengine.tools.development.testing.LocalUserServiceTestConfig;
+import com.rtb.projectmanagementtool.user.*;
+import java.util.*;
+import javax.servlet.*;
+import javax.servlet.http.*;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.*;
+
+public class AuthOpsTest {
+
+  private HttpServletRequest request;
+  private HttpServletResponse response;
+  private Cookie cookie;
+  private AuthOps auth;
+
+  private final LocalServiceTestHelper helper =
```

Comment:
```
private final LocalServiceTestHelper helper =
      new LocalServiceTestHelper(
              new LocalUserServiceTestConfig().setOAuthUserId("abc"),
              new LocalDatastoreServiceTestConfig()
                  .setAutoIdAllocationPolicy(
                      LocalDatastoreService.AutoIdAllocationPolicy.SEQUENTIAL))
          .setEnvIsLoggedIn(true);
```

Reply:
method `setUserServiceAuthInfo()` you can call in your tests to change the login status for testing